New API doc changes (#2935)
timescale/docs: api/add_retention_policy.md

|Name|Type|Description|
|-|-|-|
|`if_not_exists`|BOOLEAN|Set to `true` to avoid an error if the `drop_chunks_policy` already exists. A notice is issued instead. Defaults to `false`.|

> |`drop_created_before`|INTERVAL|Specification of cut-off point where any chunks with creation time older than this timestamp should be dropped. The cut-off point is computed as `now() - drop_created_before`. Defaults to `NULL`. Not supported for continuous aggregates yet.|

Reviewer: ```suggestion
|`drop_created_before`|INTERVAL|Chunks with creation time older than this are dropped. The cut-off point is computed as `now() - drop_created_before`. Defaults to `NULL`. Not supported for continuous aggregates.|
```

Author: done

SELECT add_retention_policy('conditions', BIGINT '600000');
```

Create a data retention policy to discard chunks created before 6 months:

```sql
SELECT add_retention_policy('conditions', drop_created_before => INTERVAL '6 months');
```

> 

Reviewer: You should probably elaborate a little here as well. Give the chunk interval for the table and explain what range of data that will be dropped.

Author: Again, chunk interval and creation times are not related. In events use cases where serial id or integer-like ranges are used, we use the creation time metadata where applicable.

Reviewer: I still think some sort of example here might make the situation a little clearer, but it is not a blocker.
